TestGenerateToolingFeed/HelperTests.cs

```diff
@@ -0,0 +1,187 @@
+using System;
+using System.Collections.Generic;
+using Newtonsoft.Json.Linq;
+using GenerateToolingFeed;
+using System.IO.IsolatedStorage;
+using Moq;
+using Newtonsoft.Json;
+
+namespace TestGenerateToolingFeed.V4Format
+{
+    public class HelperTests
+    {
+        [Theory]
+        [InlineData("Microsoft.Azure.Functions.Worker.ItemTemplates.NetCore", 4, "https://www.nuget.org/api/v2/package/Microsoft.Azure.Functions.Worker.ItemTemplates.NetCore/4.0.2945")]
+        [InlineData("Microsoft.Azure.Functions.Worker.ProjectTemplates", 4, "https://www.nuget.org/api/v2/package/Microsoft.Azure.Functions.Worker.ProjectTemplates/4.0.2945")]
+        [InlineData("Microsoft.Azure.WebJobs.ItemTemplates", 3, "https://www.nuget.org/api/v2/package/Microsoft.Azure.WebJobs.ItemTemplates/3.1.2945")]
+        [InlineData("Microsoft.Azure.WebJobs.ProjectTemplates", 3, "https://www.nuget.org/api/v2/package/Microsoft.Azure.WebJobs.ProjectTemplates/3.1.2945")]
+        [InlineData("Microsoft.Azure.Functions.Worker.ItemTemplates.NetFx", 4, "https://www.nuget.org/api/v2/package/Microsoft.Azure.Functions.Worker.ItemTemplates.NetFx/4.0.2945")]
+        [InlineData("Microsoft.Azure.Functions.Worker.ItemTemplates", 4, "https://www.nuget.org/api/v2/package/Microsoft.Azure.Functions.Worker.ItemTemplates/4.0.2945")]
+        public void Test_GetTemplateUrl(string packageId, int cliMajor, string expectedVersion)
+        {
+            var version = Helper.GetTemplateUrl(packageId, cliMajor);
+            Assert.Equal(expectedVersion, version);
+        }
+
+        [Theory]
+        [InlineData("https://raw.githubusercontent.com/Azure/azure-functions-tooling-feed/main/cli-feed-v4.json", "v4")]
+        [InlineData("https://raw.githubusercontent.com/Azure/azure-functions-tooling-feed/main/cli-feed-v4.json", "v4-prerelease")]
+        public void Test_GetReleaseVersionFromTag(string url, string tag)
+        {
+            using (var httpClient = new HttpClient())
+            {
+                var json = httpClient.GetStringAsync(url).GetAwaiter().GetResult();
+                var feed = JObject.Parse(json);
+                var resultVersion = Helper.GetReleaseVersionFromTag(feed, tag);
+
+                Assert.True(resultVersion != null);
+            }
+        }
+        
+        [Theory]
+        [InlineData("net6-isolated", "foo-net6-isolated-item", "foo-net6-isolated-project")]
+        [InlineData("netfx-isolated", "foo-netfx-isolated-item", "foo-netfx-isolated-project")]
+        public void Test_UpdateDotnetTemplatesToLatest(string dotnetVersion, string updatedItem, string updatedProject)
```

Comment:
Can you add more descriptive name on what is being tested and what is there result or a comment above the test with a summary.

Reply:
I am not sure what is being tested here, UpdateDotnetTemplatesToLatest is not called.